CHANGELOG.rst

```diff
@@ -1,6 +1,106 @@
+- Search improvements. MLflow 1.0 includes additional support for searching runs within a single experiment or a group of experiments through UI and API. The search filter API supports a simplified version of the SQL WHERE clause. In addition to searching using run's metrics and params, the API has been enhanced to support some run attributes and user and `system tags <https://mlflow.org/docs/latest/tracking.html#system-tags>`_. For details see `Search syntax <https://mlflow.org/docs/latest/search-syntax.html#syntax>`_ and `examples for programmatically searching runs <https://mlflow.org/docs/latest/search-syntax.html#programmatically-searching-runs>`_. (#1245, #1272, #1323, #1326, @mparkhe; #1052, @Zangr; #1363, @aarondav)
+- Windows support for MLflow Tracking. The Tracking portion of the MLflow client is now supported on Windows. (#1171, @eedeleon, @tomasatdatabricks)
+
+Breaking changes:
+
+Some of the breaking changes involve database schema changes. If your database instance's schema is not up-to-date, MLflow will issue an error at the start-up of ``mlflow server`` or ``mlflow ui``. To migrate an existing database to the newest schema, you can use the ``mlflow db upgrade`` CLI command. (#1155, #1371, @smurching; #1360, @aarondav)
+
+- API changes
+
+  - [Runs] Rename ``run_uuid`` to ``run_id`` in Python, Java, and REST API (#1187, @aarondav)
+  - [Runs] Remove deprecated ``RunInfo`` properties from ``start_run`` (#1220, @aarondav)
+  - Remove deprecated fields from REST, Python, and R APIs for 1.0 (#1188, @aarondav)
+  - [Models][Python] Require arguments as keywords in `log_model`, `save_model` and `add_to_model` methods in the `tensorflow` and `mleap` modules to avoid breaking changes in the future (#1226, @sueann)
+  - [Models/Deploy][Python/R] Introduce ``pyfunc.load_model`` to be consistent with other Models modules. ``pyfunc.load_pyfunc`` will be deprecated in the near future. (#1222, @sueann)
+  - Also remove the unsupported ``jars`` argument from ```spark.log_model`` (#1222, @sueann)
+  - [Projects] Remove and rename some ``projects.run`` parameters for generality and consistency. (#1222, @sueann)
+  - [Models] Rename ``dst_path`` parameter in ``pyfunc.save_model`` to ``path`` (#1221, @aarondav)
+  - [CLI] Unify ``mlflow pyfunc`` and ``mlflow rfunc`` commands as ``mlflow models`` (#1257, @tomasatdatabricks; #1321, @dbczumar)
```

Comment:
It might be good to list this earlier (maybe even in the new features part) since more users come across it. Or maybe we can organize the changes by command-line interface, programmatic APIs, and REST API somehow to make these easier to find. Some of the things mentioned here only affect people who want to use the REST API directly, but they're interspersed with things that most users will need to hear about and will be confused by if they can't see them in one place.